src/python/pants/engine/internals/scheduler.py

```diff
@@ -152,6 +155,9 @@ def __init__(
         # Create the native Scheduler and Session.
         types = PyTypes(
             paths=Paths,
+            path_metadata=PathMetadata,
```

Comment:
Not for this change obviously, but we really need a better way to register these types than to list them all in one place.

Reply:
I was able to at least remove the `PathMetadata` type from here since it is not exported from the Rust side through pyo3.